test/ios-project-service.ts

```diff
@@ -159,3 +159,69 @@ describe("Cocoapods support", () => {
 		});
 	}
 });
+
+describe("Static libraries support", () => {
+	if (require("os").platform() !== "darwin") {
+		console.log("Skipping static library tests. They cannot work on windows.");
```

Comment:
You may elect to return here and omit the else clause. This way, the body of the tests will be one tab to the left and much easier to read.